provider/openstack/networking.go

```diff
@@ -425,6 +425,6 @@ func noNetConfigMsg(err error) string {
 		err.Error())
 }
 
-func (n *NeutronNetworking) NetworkInterfaces(instId instance.Id) ([]network.InterfaceInfo, error) {
+func (n *NeutronNetworking) NetworkInterfaces(ids []instance.Id) ([][]network.InterfaceInfo, error) {
```

Comment:
This looks like it could be problem waiting for us. Why is this not supported?

Reply:
Yes; we will absolutely need something here in order for the new instance-poller to work, but I think this can be landed for business-as-usual.